### PR TITLE
None position bugfix

### PR DESCRIPTION
**Pull request by szok**

### CODE REVIEW

**Comment by mkurek** [comment]
imo we should provide default for position field (0?) and make it PositiveIntegerField - @ar4s what do you think?

File: src/ralph/data_center/models/physical.py [hunk 1]
```diff
@@ -214,9 +214,11 @@ def get_free_u(self):
 
         def fill_u_list(objects, height_of_device=lambda obj: 1):
             for obj in objects:
-                start = obj[0] - 1
+                position = obj[0] or 0
```
Comment (ghost) [inline src/ralph/data_center/models/physical.py]: For me fix should be simply:

```
for obj in objects:
    if obj[0] == None:
        continue  # no position means do not take place in rack
```

Comment (ar4s) [inline src/ralph/data_center/models/physical.py]: :+1:, but I think it should be `if obj[0]` for `0` and `None`

**Comment by ar4s** [comment]
@mkurek I agree with you it should be `PositiveIntegerField` and `0` as a default value.

**Comment by ghost** [comment]
Is 0 valid value, like 1, 2 or 3?

File: src/ralph/data_center/models/physical.py [hunk 2]
```diff
@@ -214,6 +214,8 @@ def get_free_u(self):
 
         def fill_u_list(objects, height_of_device=lambda obj: 1):
             for obj in objects:
+                if not obj[0]:
```
Comment (mkurek) [inline src/ralph/data_center/models/physical.py]: the question here is if object with position 0 should (or could) have non-zero height: https://github.com/allegro/ralph/blob/ng/src/ralph/data_center/models/physical.py#L333 ? Anyone know something about it?

Comment (mkurek) [inline src/ralph/data_center/models/physical.py]: after discussion with @xliiv:
- make this condition more explicit: `if obj[0] == 0 or obj[0] is None`
- add some comment that position is None when objects simply does not have (assigned) position and position 0 is for some accessories (pdu?) with left-right orientation and should not be included in free/filled space.

Comment (ghost) [inline src/ralph/data_center/models/physical.py]: +1

**Comment by mkurek** [comment]
:ok:

**Comment by ghost** [comment]
:ok:

**Pull request merged**